java/src/main/java/com/powsybl/python/security/SecurityAnalysisCFunctions.java

```diff
@@ -164,14 +181,20 @@ private static void setSecurityAnalysisResultPointer(PyPowsyblApiHeader.Continge
         contingencyPtr.limitViolations().setPtr(limitViolationPtr);
     }
 
-    private static PyPowsyblApiHeader.ArrayPointer<PyPowsyblApiHeader.ContingencyResultPointer> createContingencyResultArrayPointer(SecurityAnalysisResult result) {
+    private static PyPowsyblApiHeader.PreContingencyResultPointer createPreContingencyResultArrayPointer(SecurityAnalysisResult result) {
+        int resultCount = result.getPostContingencyResults().size() + 1; // + 1 for pre-contingency result
```

Comment:
just one?

Reply:
I forget to remove it now that they are separated